cmd/mark-and-sweep-stale-issues/mark_and_sweep_stale_issues.go

```diff
@@ -35,21 +35,23 @@ var (
 
 	// All the repos to apply apply these to.
 	defaultRepos = []repo{
-		{"jekyll", "jekyll"},
-		{"jekyll", "jekyll-admin"},
-		{"jekyll", "jekyll-import"},
-		{"jekyll", "github-metadata"},
-		{"jekyll", "jekyll-redirect-from"},
-		{"jekyll", "jekyll-feed"},
-		{"jekyll", "jekyll-compose"},
-		{"jekyll", "jekyll-watch"},
-		{"jekyll", "jekyll-seo-tag"},
-		{"jekyll", "jekyll-sitemap"},
-		{"jekyll", "jekyll-sass-converter"},
-		{"jekyll", "jemoji"},
-		{"jekyll", "jekyll-gist"},
-		{"jekyll", "jekyll-coffeescript"},
-		{"jekyll", "plugins"},
+		repo{"jekyll", "jekyll"},
+		repo{"jekyll", "jekyll-admin"},
+		repo{"jekyll", "jekyll-import"},
```

Comment:
In Go, you don't need to re-specify the type for members of an array literal. I already specified `[]repo{}` on line 37, so it knows everything inside should be a repo, too. Hence just `{"owner", "name"}`. Can you remove the `repo` prefix on all these lines, please?